### ISSUE TITLE
Don't render subway icon when disused=yes

### ISSUE THREAD

**Issue by Vonter** [organicmaps/organicmaps#4138]
Under constructions stations or stations which are not currently operational should not have the subway icon rendered on them. For example, [this station, which has `disused=yes`](https://www.openstreetmap.org/node/9953937121) is rendered with the custom subway icon with the `2022.12.18-12-FDroid` version. However, this station is non operational (actually under construction) so showing a subway symbol can be misleading.

**Comment by endim8** [comment]
also see https://github.com/organicmaps/organicmaps/issues/1038, we need to figure out how to properly handle disused pois

**Comment by Vonter** [comment]
I agree, using the lifecycle prefixes for non-operational stations makes more sense to me. I've made this change for the under-construction/unused stations in the city that I originally found the issue in.
